src/ToDoList/ToDoList.tsx

```diff
@@ -1,35 +1,56 @@
 import React from "react";
 import ToDoItem from "../ToDoItem/ToDoItem";
 import { TodoItem } from "../types";
-import Box from "@mui/material/Box";
+import { Container, Box } from "@mui/material";
+import { Droppable } from "react-beautiful-dnd";
+import { DragDropContext } from "react-beautiful-dnd";
+import { resourceUsage } from "process";
+import type { DroppableProvided } from "react-beautiful-dnd";
 
 interface ToDoListProps {
   items: TodoItem[];
   onDelete: (id: number) => void;
   onToggle: (id: number) => void;
+  onDragEnd: (result: any) => void;
 }
 
-const ToDoList: React.FC<ToDoListProps> = ({ items, onDelete, onToggle }) => {
+const ToDoList: React.FC<ToDoListProps> = ({
+  items,
+  onDelete,
+  onToggle,
+  onDragEnd,
+}) => {
   return (
-    <Box
-      sx={{
-        display: "flex",
-        flexDirection: "column",
-        alignItems: "center",
-        rowGap: 2,
```

Comment:
You cannot use `rowGap` for creating space between items.

That's because after you start dragging any item that space disappears and the whole list shifts. It's better to use `margin-bottom` on each item instead.